core/controllers/access_validators.py

```diff
@@ -229,3 +229,41 @@ def get(self) -> None:
 
         if not blog_post:
             raise self.PageNotFoundException
+
+
+class BlogAuthorProfilePageAccessValidationHandler(
+    base.BaseHandler[Dict[str, str], Dict[str, str]]
+):
+    """Validates whether request made to correct blog author page route."""
```

Comment:
Can you make this clearer? I don't understand it.

Doesn't this validate access permissions rather than whether the request is made to the correct place?

Reply:
Changed to 'Validates access to blog author profile page'. Sorry.